### YamlMime:PythonClass
uid: corehttp.exceptions.SerializationError
name: SerializationError
fullName: corehttp.exceptions.SerializationError
module: corehttp.exceptions
inheritances:
- builtins.ValueError
summary: Raised if an error is encountered during serialization.
constructor:
  syntax: SerializationError()
methods:
- uid: corehttp.exceptions.SerializationError.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: corehttp.exceptions.SerializationError.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: corehttp.exceptions.SerializationError.args
  name: args
